PMM-10457 Moved docs from pmm repo (#341)
percona-platform/dbaas-controller: README.md

[![codecov.io Code Coverage](https://codecov.io/gh/percona-platform/dbaas-controller/branch/main/graph/badge.svg)](https://codecov.io/github/percona-platform/dbaas-controller?branch=main)
[![CLA assistant](https://cla-assistant.percona.com/readme/badge/percona-platform/dbaas-controller)](https://cla-assistant.percona.com/percona-platform/dbaas-controller)

DBaaS Controller exposes a simplified API for managing Percona Kubernetes Operators.

## How to get involved

We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are.

If you’re looking for information about how you can contribute, we have [contribution guidelines](CONTRIBUTING.md) across all our repositories in `CONTRIBUTING.md` files. Some of them may just link to the main project’s repository’s contribution guidelines.

We're looking forward to your contributions and hope to hear from you soon on our [Forums](https://forums.percona.com) and [Discord](https://per.co.na/discord).

## Submitting Bug Reports

> If you find a bug in Percona Monitoring and Management  or one of the related projects, you should submit a report to that project's [JIRA](https://jira.percona.com) issue tracker. Some of related project also have GitHub Issues enabled, so you also could submit there.

Reviewer: ```suggestion
If you find a bug in Percona Monitoring and Management  or one of the related projects, you can submit a report to that project's [JIRA](https://jira.percona.com) issue tracker. Some of the related projects also have GitHub Issues enabled, so you can also submit there.
```
